SPECS-EXTENDED/sblim-cim-client2/sblim-cim-client2.spec

```diff
@@ -48,6 +49,7 @@ Manual and sample code for %{name}.
 
 %prep
 %setup -q -n %{project_folder}
+%patch0 -p1 -b .fix-for-java-11-openjdk
```

Comment:
how about we use `%autosetup` ?

Reply:
Same answer as with your previous question. We'll do a thorough clean-up when this is moved to `SPECS`. I didn't want to muddy the water too much, since the PR was already getting big.